docs/Developer-Guide.md

```diff
@@ -212,10 +212,14 @@ $ sudo systemctl restart systemd-journald
 >
 > - You should only do this step if you are testing with the latest version of the agent.
 
-The rust-agent is built with a static linked `musl.` To configure this:
+The rust-agent is built with a static linked `musl.` The default rust component used is `musl`, but on `ppc64le` and `s390x`,
+`gnu` should be used. To configure this:
```

Comment:
Thanks, very kind @YchauWang! I think we can do some grammar fixes while we're here:
```suggestion
The agent is built with a statically linked `musl.` The default libc used is `musl`, but on `ppc64le` and `s390x`,
`gnu` should be used. To configure this:
```

(we haven't supported the Go agent in a long time, I think it's time to stop calling it "the Rust agent" like that's something extraordinary.)

Reply:
Thanks, @Jakob-Naucke.